src/modules/link-selector.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Link selector
+ * @author Deux Huit Huit
+ */
+(function ($) {
+
+	'use strict';
+
+	var scope = $('body');
+	var fakeAnchor = $('<a />');
+
+	var update = function () {
+		var currentPath = window.location.pathname;
+		scope.find('[href]').each(function () {
```

Comment:
We should scope the behavior with a js-class. This will avoid parsing external injected DOM.

Reply:
Then add a way like with a data attribute to disable the feature. Also your selector need more precision. it will break if someone add a `<map><area` href="" `/></map>` node and maybe other tags.

Reply:
```suggestion
		scope.find('a[href]').each(function () {
```